docs/moment/04-displaying/06-calendar-time.md

```diff
@@ -54,4 +54,19 @@ moment().calendar(null, {
     sameElse: 'DD/MM/YYYY'
 });
 ```
-`sameElse` is used as the format when the moment is more than a week away from the `referenceTime`
\ No newline at end of file
+`sameElse` is used as the format when the moment is more than a week away from the `referenceTime`
+
+**Note**: From version **2.14.0** the formats argument to calendar can be
+a callback that is executed within the moment context with a single argument
+now:
+
+```javascript
+moment().calendar(null, {
+  sameDay: function (now) {
+    if (this.before(now)) {
+      return '[Will Happen Today]';
+    } else {
+      return '[Happened Today]';
+    }
+    /* ... */
+});
```

Comment:
I think this example is missing the closing back ticks, hence the build not running properly.